src/client/components/HealthCareApp.jsx

```diff
@@ -158,20 +158,23 @@ class HealthCareApp extends React.Component {
     if (submissionStatus === false) {
       submitButton = (
         <ProgressButton
+            additionalClass="submit-status-not-submitted"
```

Comment:
Gotcha. I think adding additional classes should be a last resort, so let's see if we can target these instead by checking the text of the button.